assembl/static2/js/app/components/administration/saveButton.jsx

```diff
@@ -103,7 +104,9 @@ export const getMutationsPromises = (params: Params) => {
 };
 
 export const DumbSaveButton = ({ disabled, saveAction, specificClasses }: Props) => {
-  const buttonClasses = specificClasses || 'save-button button-submit button-dark right';
+  const isHoverClass = !disabled ? 'button-dark' : '';
+  const buttonClasses = specificClasses || classNames('save-button button-submit right', isHoverClass);
```

Comment:
You can also do `classNames({specificClasses: !!isHoverClass}, 'save-button button-submit right', isHoverClass)`, it's kind of the point, no?

Reply:
@ayyazdaniaryan won't be able to applied your refactor since `specificClasses?: ?string`